fuzz/json_array_fuzzer.cc

```diff
@@ -0,0 +1,28 @@
+#include <fuzzer/FuzzedDataProvider.h>
+
+#include "json.h"
+#include "json_tokener.h"
+
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
+	FuzzedDataProvider fdp(data, size);
+	json_object *my_array = json_object_new_array();
+	for (int i = 0; i < 3; ++i) {
+		json_object *jso = json_tokener_parse(fdp.ConsumeRandomLengthString(10).c_str());
+		if (jso == NULL) {
+			continue;
+		}
+		json_object_array_add(my_array, jso);
+	}
+	json_object_array_insert_idx(my_array, fdp.ConsumeIntegralInRange<size_t>(0, 10),
+	                             json_object_new_int(fdp.ConsumeIntegral<int>()));
+	json_object_array_get_idx(my_array, fdp.ConsumeIntegralInRange<size_t>(0, 10));
```

Comment:
Just to clarify, have we verified that these functions are called? I just want to make sure that they are not optimized away by the compiler since we are not using the return value